src/dpi.rs

```diff
@@ -28,10 +28,20 @@
 //! feel like they have bad eyesight, which will potentially cause them to think about growing elderly, resulting in
 //! them entering an existential panic. Once users enter that state, they will no longer be focused on your application.
 //!
-//! There are two ways to get the DPI factor: either by calling
-//! [`MonitorId::get_hidpi_factor`](../struct.MonitorId.html#method.get_hidpi_factor), or
-//! [`Window::get_hidpi_factor`](../struct.Window.html#method.get_hidpi_factor). You'll almost always use the latter,
-//! which is basically equivalent to `window.get_current_monitor().get_hidpi_factor()` anyway.
+//! There are two ways to get the DPI factor:
+//! - You can track the `WindowEvent::HiDpiFactorChanged` event of your windows. This event is sent any
+//!   time the HiDPI factor changes, be it because the window moved to an other monitor, or the
```

Comment:
This is the only time in the docs that "HiDPI factor" is said instead of "DPI factor", so it could cause people to assume there's a distinction... though, arguably I should've been writing "HiDPI factor" the whole time, since that corresponds to the actual API naming.